cmd/utils.go

```diff
@@ -1003,14 +1003,15 @@ func getLatestVersion() string {
 
 func doVersionCheck(data []byte, old string, new string, file string) {
 	var latest = getLatestVersion()
-	if VERSION != "vlatest" && VERSION != latest {
+	if latest != "none" && VERSION != "vlatest" && VERSION != latest {
 		Info.log("*\n*\n*\n\nA new CLI update is available.\nPlease go to " + LatestVersionURL + " and update from " + VERSION + " --> " + latest + ".\n\n*\n*\n*")
```

Comment:
I wonder if we should link to https://appsody.dev/docs/getting-started/installation

Actually it would be really nice to detect the client's OS and give specific instructions. For example, on mac we could say "Please run `brew upgrade appsody`."

Reply:
Done, although I am unsure how specific we want to be with our instructions

Reply:
Agree, see new comment below.